encoding/json/register.go

```diff
@@ -37,44 +37,44 @@ var (
 	_interfaceEmptyType = reflect.TypeOf((*interface{})(nil)).Elem()
 )
 
-// Register calls the Registrar's Register method.
+// Register calls the RouteTable's Register method.
 //
 // This function exists for backwards compatibility only. It will be removed
 // in a future version.
 //
-// Deprecated: Use the Registrar's Register method directly.
-func Register(r transport.Registrar, rs []transport.Registrant) {
+// Deprecated: Use the RouteTable's Register method directly.
+func Register(r transport.RouteTable, rs []transport.Procedure) {
 	r.Register(rs)
 }
 
-// Procedure builds a Registrant from the given JSON handler. handler must be
+// Procedure builds a Procedure from the given JSON handler. handler must be
 // a function with a signature similar to,
 //
 // 	f(ctx context.Context, reqMeta yarpc.ReqMeta, body $reqBody) ($resBody, yarpc.ResMeta, error)
 //
 // Where $reqBody and $resBody are a map[string]interface{} or pointers to
 // structs.
-func Procedure(name string, handler interface{}) []transport.Registrant {
-	return []transport.Registrant{
+func Procedure(name string, handler interface{}) []transport.Procedure {
```

Comment:
It occurs to me that these are building a list of Procedures, not a single
Procedure. (That's so that you can pass the output of Procedure directly into
Register.)  We should probably pluralize the mention of the `Procedure` type in the godocs above.